src/styles/index.scss

```diff
@@ -1,3 +1,58 @@
+@import './utils/variables';
+@import 'block/header';
+@import 'block/pruduct-card';
```

Comment:
Typo in the file name import: 'pruduct-card' should be 'product-card'. Make sure the file is named correctly.